context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+Context Free is useful for people who are interested in creating
+computer-generated art.  This could have uses for graphical designs or just
+expressing creativity.  Other features in Context Free could interest animators
```

Comment:
Good point, perhaps "animators" is a bit strong in the sense that you probably won't be making any feature movies with Context Free.  I more meant people who are interested in a specific set of minor changes to their shape over time.  From the documentation, I saw that one of the shape adjustments you can include is the "lifetime" of the shape, so that you can control when it appears and disappears.  I didn't explore this feature, since we were only submitting a .png, but I was thinking that you could get some neat effects by specifying when objects enter and exit the screen.  Other animations, like moving the objects, seem a lot more difficult in Context Free.